src/isEmpty/index.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Checks if an array is empty or not
+ *
+ * @param array - The array to check for
+ * @returns A boolean If the given array is empty or not
+ *
+ * @public
+ */
+export default function isEmpty<T>(val: T[]): boolean
```

Comment:
Please give type arguments meaningful names (here and everywhere in the PR):

```suggestion
export default function isEmpty<Type>(val: Type[]): boolean
```